crossfire/__init__.py

```diff
@@ -57,20 +57,24 @@ def occurrences(
     )
 
 
+def _make_flatten(data, keys):
```

Comment:
I have separated this parto considering the idea of DRY.
and, ofcourse, considering that, for now, we will keep converting `DataFrame` to `dict`.

Reply:
I usually think that prefixes like `get,` `load,` and `make` hardly ever contribute to the semantics of the code. If you call a function, it might be implied that it is actioning (getting something, making something, loading something). Thus, if `flatten` is the primary function and you want part of it to be more _internal_, I would use `_flatten` for the “auxiliary” function.

Reply:
Good point. I have changed this auxiliary function back to `flatten` function and created anotherone for `DataFrame`, called `flatten_df`. I will change it to `_flatten_df`,  now.